src/pages/dashboard/widgets/instance-hours/index.tsx

```diff
@@ -48,6 +48,18 @@ function InstanceHoursContent() {
         filterPlaceholder: 'Filter instance types',
         xTickFormatter: dateFormatter,
       }}
+      detailPopoverSeriesContent={({ series, y }) => ({
+        key: series.title,
+        value: (
+          <Link
+            external={true}
+            href="#"
+            ariaLabel={`See details for ${y} hours on ${series.title} (opens in a new tab)`}
```

Comment:
`opens a new tab` is already provided by i18n provider https://github.com/cloudscape-design/components/blob/main/src/i18n/messages/all.en.json#L147

Reply:
Not if you use the `ariaLabel` prop, because then the ARIA label masks all content inside the link, including the icon, for the screen readers.